Server/JokesOnYou.Web.Api/Services/UserService.cs

```diff
@@ -26,24 +26,27 @@ public UserService(IUserRepository userRepository, SignInManager<User> signInMan
             _tokenService = tokenService;
         }
 
-        public async Task DeleteUser(string id)
+        public async Task<IEnumerable<AllUsersDTO>> GetAll()
         {
-            throw new NotImplementedException();
+            var allUsers = await _userRepository.GetAllUsersAsync();
+            return allUsers;
         }
 
-        public async Task<IEnumerable<User>> GetAll()
+        public async Task DeleteUser(string id)
         {
-            throw new NotImplementedException();
-        }
+            var user = await _userRepository.GetUserAsync(id) ?? null;
 
-        public async Task<User> GetUserById(string id)
-        {
-            throw new NotImplementedException();
+            if (user == null)
+                throw new Exception();
+
+            var userToDelete = new User { Id = user.Id };
```

Comment:
Whats the reason for getting a user and then creating a new object out of that user?